ggsdk/sdk.py

```diff
@@ -0,0 +1,26 @@
+import typing
+
+from ggcore.config import SdkBootstrapConfig
+from ggcore.core import SdkCore
+
+
+class GraphGridSdk:
+    """
+    GraphGridSdk is the user-facing resource to interact with GraphGrid modules
```

Comment:
Couple of things here, that also apply to other doc strings in this PR:

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

I.e. this should be a command
> For one liner docstrings, please keep the closing """ on the same line:

I.e. the newlines should be removed for this one line doc string

Reply:
@HarryDunham Thank you, appreciate the comment and direction here. I will update this and other doc strings